include/avif/avif.h

```diff
@@ -551,7 +551,17 @@ typedef enum avifRGBFormat
     AVIF_RGB_FORMAT_ARGB,
     AVIF_RGB_FORMAT_BGR,
     AVIF_RGB_FORMAT_BGRA,
-    AVIF_RGB_FORMAT_ABGR
+    AVIF_RGB_FORMAT_ABGR,
+    // RGB_565 format uses five bits for the red and blue components and six
+    // bits for the green component. Each RGB pixel is 16 bits (2 bytes) are
+    // packed as follows:
+    //   Byte 1: [r4 r3 r2 r1 r0 g5 g4 g3]
+    //   Byte 2: [g2 g1 g0 b4 b3 b2 b1 b0]
```

Comment:
I suggest we describe this as a uint16_t value in memory instead of Byte 1 and Byte 2:

```
    //   uint16_t: [r4 r3 r2 r1 r0 g5 g4 g3 g2 g1 g0 b4 b3 b2 b1 b0]
```

This matches the code:

```
        *(uint16_t *)ptrR = (uint16_t)((B >> 3) | ((G >> 2) << 5) | ((R >> 3) << 11));
```

The reason "Byte 1" and "Byte 2" are confusing is that they seem to depend on endianness. On little-endian computers (all computers we actually have access to today), Byte 1 and Byte 2 are in the following order in memory:

```
    //   Byte 1 (lower address):  [g2 g1 g0 b4 b3 b2 b1 b0]
    //   Byte 2 (higher address): [r4 r3 r2 r1 r0 g5 g4 g3]
```

By describing RGB_565 pixels as uint16_t values, we avoid the byte order issue. (This is why I said that we could consider RGB_565 as having one channel of bit depth 16.)

Reply:
Done.